BUG: CI is failing due the new continuumio/miniconda3:latest

```sh
Status: Downloaded newer image for continuumio/miniconda3:latest
 ---> 83375d57598f
Step 2/10 : RUN apt-get -qq update --yes  && apt-get -qq install --yes --no-install-recommends     build-essential git make clang libboost-dev postgresql-client ca-certificates openjdk-8-jre  && rm -rf /var/lib/apt/lists/*
 ---> Running in 7fc6867a735d
E: Unable to locate package openjdk-8-jre‌
Service &#x27;ibis&#x27; failed to build: The command &#x27;&#x2F;bin&#x2F;sh -c apt-get -qq update --yes  &amp;&amp; apt-get -qq install --yes --no-install-recommends     build-essential git make clang libboost-dev postgresql-client ca-certificates openjdk-8-jre  &amp;&amp; rm -rf &#x2F;var&#x2F;lib&#x2F;apt&#x2F;lists&#x2F;*&#x27; returned a non-zero code: 100‌
##[error]Bash exited with code '1'.
```

--- a/ci/Dockerfile.dev
+++ b/ci/Dockerfile.dev
@@ -2,7 +2,7 @@ FROM continuumio/miniconda3
 
 RUN apt-get -qq update --yes \
  && apt-get -qq install --yes --no-install-recommends \
-    build-essential git make clang libboost-dev postgresql-client ca-certificates openjdk-11-jre \
+    build-essential git make clang libboost-dev postgresql-client ca-certificates \
  && rm -rf /var/lib/apt/lists/*
 
 ARG PYTHON

--- a/ci/Dockerfile.dev
+++ b/ci/Dockerfile.dev
@@ -8,13 +8,13 @@ RUN apt-get -qq update --yes \
 ARG PYTHON
 ADD ci/requirements-$PYTHON-dev.yml /
 
-RUN conda config --add channels conda-forge \
-  && conda update --all --yes --quiet \
-  && conda env create --name ibis-env --file /requirements-$PYTHON-dev.yml \
-  && conda install --yes conda-build \
-  && conda clean --all --yes
+RUN /opt/conda/bin/conda config --add channels conda-forge \
+  && /opt/conda/bin/conda update --all --yes --quiet \
+  && /opt/conda/bin/conda env create --name ibis-env --file /requirements-$PYTHON-dev.yml \
+  && /opt/conda/bin/conda install --yes conda-build \
+  && /opt/conda/bin/conda clean --all --yes
 
-RUN echo 'source activate ibis-env && exec "$@"' > activate.sh
+RUN echo 'source /opt/conda/bin/activate ibis-env && exec "$@"' > activate.sh
 
 COPY . /ibis
 WORKDIR /ibis

--- a/ci/Dockerfile.dev
+++ b/ci/Dockerfile.dev
@@ -2,7 +2,7 @@ FROM continuumio/miniconda3
 
 RUN apt-get -qq update --yes \
  && apt-get -qq install --yes --no-install-recommends \
-    build-essential git make clang libboost-dev postgresql-client ca-certificates openjdk-8-jre \
+    build-essential git make clang libboost-dev postgresql-client ca-certificates openjdk-11-jre \
  && rm -rf /var/lib/apt/lists/*
 
 ARG PYTHON

--- a/ci/Dockerfile.docs
+++ b/ci/Dockerfile.docs
@@ -3,7 +3,7 @@ FROM ibis:$PYTHON
 
 # fonts are for docs
 RUN apt-get -qq update --yes \
- && apt-get -qq install --yes ttf-dejavu \
+ && apt-get -qq install --yes ttf-dejavu iputils-ping \
  && rm -rf /var/lib/apt/lists/*
 
 ADD ci/requirements-docs.yml /

--- a/ci/Dockerfile.docs
+++ b/ci/Dockerfile.docs
@@ -8,12 +8,12 @@ RUN apt-get -qq update --yes \
 
 ADD ci/requirements-docs.yml /
 
-RUN conda config --add channels conda-forge \
- && conda update --all --yes \
- && conda install --name ibis-env --yes --file /requirements-docs.yml \
- && conda clean --all --yes
+RUN /opt/conda/bin/conda config --add channels conda-forge \
+ && /opt/conda/bin/conda update --all --yes \
+ && /opt/conda/bin/conda install --name ibis-env --yes --file /requirements-docs.yml \
+ && /opt/conda/bin/conda clean --all --yes
 
-RUN echo 'source activate ibis-env && exec "$@"' > activate.sh
+RUN echo 'source /opt/conda/bin/activate ibis-env && exec "$@"' > activate.sh
 
 COPY . /ibis
 WORKDIR /ibis

--- a/ci/azure/windows.yml
+++ b/ci/azure/windows.yml
@@ -48,7 +48,7 @@ jobs:
     - script: conda update --all conda=$(conda.version)
       displayName: 'Update conda and install an appropriate version'
 
-    - script: conda create --name $(conda.env) python=$(python.version) numpy pandas pytables ruamel.yaml jinja2 pyarrow multipledispatch pymysql sqlalchemy psycopg2 graphviz click mock plumbum geopandas toolz regex
+    - script: conda create --name $(conda.env) python=$(python.version) numpy pandas pytables ruamel.yaml jinja2 pyarrow multipledispatch pymysql "sqlalchemy<1.3.6" psycopg2 graphviz click mock plumbum geopandas toolz regex
       displayName: 'Create conda environment'
 
     - script: |

--- a/ci/requirements-3.5-dev.yml
+++ b/ci/requirements-3.5-dev.yml
@@ -21,6 +21,7 @@ dependencies:
   - multipledispatch>=0.6.0
   - mypy
   - numpy>=1.11
+  - openjdk=8
   - pandas>=0.21
   - pip
   - plumbum

--- a/ci/requirements-3.5-dev.yml
+++ b/ci/requirements-3.5-dev.yml
@@ -40,7 +40,7 @@ dependencies:
   - regex
   - requests
   - ruamel.yaml
-  - sqlalchemy>=1.1
+  - sqlalchemy>=1.1,<1.3.7
   - thrift>=0.9.3
   # required for impyla in case of py3
   - thriftpy

--- a/ci/requirements-3.6-dev.yml
+++ b/ci/requirements-3.6-dev.yml
@@ -44,7 +44,7 @@ dependencies:
   - requests
   - ruamel.yaml
   - shapely
-  - sqlalchemy>=1.1
+  - sqlalchemy>=1.1,<1.3.7
   - thrift>=0.9.3
   - thriftpy2  # required for impyla in case of py3
   - toolz

--- a/ci/requirements-3.6-dev.yml
+++ b/ci/requirements-3.6-dev.yml
@@ -20,6 +20,7 @@ dependencies:
   - multipledispatch>=0.6.0
   - mypy
   - numpy>=1.11
+  - openjdk=8
   - pandas>=0.21
   - pip
   - plumbum

--- a/ci/requirements-3.7-dev.yml
+++ b/ci/requirements-3.7-dev.yml
@@ -44,7 +44,7 @@ dependencies:
   - requests
   - ruamel.yaml
   - shapely
-  - sqlalchemy>=1.1
+  - sqlalchemy>=1.1,<1.3.7
   - thrift>=0.9.3
   - thriftpy2  # required for impyla in case of py3
   - toolz

--- a/ci/requirements-3.7-dev.yml
+++ b/ci/requirements-3.7-dev.yml
@@ -20,6 +20,7 @@ dependencies:
   - multipledispatch>=0.6.0
   - mypy
   - numpy>=1.11
+  - openjdk=8
   - pandas>=0.21
   - pip
   - plumbum

